posthog/utils.py

```diff
@@ -223,7 +223,6 @@ def render_template(template_name: str, request: HttpRequest, context: Dict = {}
     # Set the frontend app context
     if not request.GET.get("no-preloaded-app-context"):
         from posthog.api.user import UserSerializer
-        from posthog.models import EventDefinition
```

Comment:
this is just a lingering unused import I found btw

Reply:
Thanks for the note!